app/config/tstvhq/rooms.json

```diff
@@ -425,10 +413,13 @@
         {
             "id": "cemetery",
             "name": "Cemetery",
-            "type": "JITSI",
-            "capacity": 10,
+            "type": "IFRAME",
+            "capacity": 15,
             "description": "There's room for all of us to rest here",
             "backgroundImage": "./https://tstvhq.s3.amazonaws.com/backgrounds/cemeterygreen.jpg",
+            "iframeOptions": {
+                "src": "https://youtu.be/ydqOaUvqhbQ"
```

Comment:
These should be youtube embed links, which can be obtained by going to the youtube video, clicking `share`, and then clicking `embed`.